what4/src/What4/Protocol/SMTLib2/Response.hs

```diff
@@ -71,6 +71,8 @@ data SMTResponse = AckSuccess
                  | AckSat
                  | AckUnsat
                  | AckUnknown
+                 | AckInfeasible
+                 | AckFail
```

Comment:
These can only arise from SyGuS sessions, correct? Worth stating in the comments if so.